executor.go

```diff
@@ -161,6 +164,8 @@ func (e *ExecutorBuilder) NewCmdExecutor(ctx context.Context) cmd.Executor {
 
 		metricCollector := e.MetricsAndTracesRegister.Register(ctx, cfg, logger)
 
+		e.WebsocketProxyRegister.Register(ctx, cfg, logger)
```

Comment:
can we make this optional ? to only start proxy web socket in case it is configured ? Or is it handled inside the proxy internally ?

Reply:
It's handled in krakend-websocket as other modules